packages/micropip/test_micropip.py

```diff
@@ -748,3 +775,13 @@ async def test_freeze(mock_fetch: mock_fetch_cls) -> None:
     assert pkg_metadata["imports"] == toplevel[0]
     assert dep1_metadata["imports"] == toplevel[1]
     assert dep2_metadata["imports"] == toplevel[2]
+
+
+@run_in_pyodide(packages=["micropip"])
+async def test_emfs_micropip(selenium_standalone_micropip, regex_wheel_name):
```

Comment:
Maybe use [snowballstemmer wheel](https://github.com/pyodide/pyodide/tree/main/packages/micropip/test) instead, to make the test little more self-contained?

Some other tests use it such as:

https://github.com/pyodide/pyodide/blob/8c8109f1dfa9f3716f6391351152bcef23ffc006/packages/micropip/test_micropip.py#L248-L251

Reply:
I'm not sure I understand. Isn't the test already self contained?